app/controllers/order_imports_controller.rb

```diff
@@ -28,6 +28,14 @@ def create
     redirect_to new_facility_order_import_path
   end
 
+  def error_report
+    if order_import.error_file_present?
+      redirect_to order_import.error_file_download_url
```

Comment:
It looks like Paperclip does its own content type detection if we don't specify it, and CSVs were seen as `text/plain`. Firefox retains the `.csv` extension, but I guess when Chrome sees `text/plain` it changes the extension. I've got a patch that should set the content type header correctly now.